src/Arcus.Messaging.Pumps.Abstractions/MessagePump.cs

```diff
@@ -44,7 +65,12 @@ protected MessagePump(IConfiguration configuration, IServiceProvider serviceProv
         /// <summary>
         /// Gets the boolean flag that indicates whether the message pump is started and receiving messages.
         /// </summary>
-        public bool IsStarted { get; private set; }
+        public bool IsStarted { get; protected set; }
```

Comment:
The comment is a bit confusing.
The pump can be started, but not receiving messages because the CircuitState is open.

Reply:
This is more closely linked to each other nd automatically assigned internally. The state is tracked by start/stoping the pump, just s this boolean. There is no 'cricuit breaker'-specific methods exposed, only the start/stop methods that internally sets the boolean and the state. The boolean here acts as a more friendly exposure of the 'state' of the pump.

Reply:
I just wanted to say that it's maybe better to remove the part `and receiving messages` ?

Reply:
Yes, that we can do.

Reply:
Removed the part here: https://github.com/arcus-azure/arcus.messaging/pull/444/commits/d5cfabf7798c51101c6dd1bfdcd44a7db74510dc